locale/de_DE.UTF-8/LC_MESSAGES/palma.po

```diff
@@ -164,12 +164,16 @@ msgid ""
 "With PalMA, you can share your documents and your desktop\n"
 "            with your learning group."
 msgstr ""
+"Mit PalMA kanst du deine Dokumente und deinen Desktop\n"
+"			mit deiner Lerngruppe teilen."
 
 #: index.php:1113
 msgid ""
 "Team members can join the group at any time. All they need\n"
 "            is URL and PIN."
 msgstr ""
+"Temmitglieder können der Gruppe jederzeit beitreten. Sie\n"
+"			brauchen nur die URL und den PIN."
```

Comment:
PIN = Persönliche Identifikationsnummer, so we need "die PIN" instead of "den PIN"